service/api_logic/user_logic.py

```diff
@@ -67,59 +74,49 @@ def __send_reset_email(self, user: User, token: str):
 
 
     def __get_reset_token(self, email) -> str:
-        user = self._user_dal.get_user_by_email_or_username(email)
+        user = self.get_user_by_email_or_username(email)
         if not user:
             raise UserDoesNotExistError(email)
 
         return self._serializer.dumps(user.username, salt = "email-confirm")
 
 
-    def reset_user_password(self, email, new_password: str) -> str:
-        user = self._user_dal.get_user_by_email_or_username(email)
+    def reset_user_password(self, email, new_password: str):
```

Comment:
Don't fix it. Maybe we'll do it in scope of another PR.

Make sure you convey that to others:

There is an order in which we place our methods within our classes. Usually it's private -> protected -> public, meaning private are on top of the class and everything else is below. 

Make sure you know your code conventions for the language you use, since they usually differ from one to another.